plugins/modules/zos_lineinfile.py

```diff
@@ -195,6 +195,9 @@
     needs to be encoded, it should be cataloged first.
   - For supported character sets used to encode data, refer to the
     L(documentation,https://ibm.github.io/z_ansible_collections_doc/ibm_zos_core/docs/source/resources/character_set.html).
+  - When working with an MVS file and ZOAU version 1.3.0, combining C(regexp)
```

Comment:
I'm not sure if `MVS file` is an official z/OS term, if it's data sets, maybe use `data sets`? Valid data sets are already defined above, so we probably don't need to re-specify them

>When working with data sets and ZOAU version 1.3.0, combining ...

Also, "throw an error" is more of a developer-y word to me, maybe we can say the "module fails or something like that?

Maybe:
> When working with ZOAU version 1.3.0, the module fails when C(src) is a data set and either C(insertafter) or C(insertbefore) is set in addition to C(regexp).

^ I like having the ZOAU version right at the beginning so if anyone is on a different version, they dont have to bother reading the rest of the sentence